odoo/addons/base/res/res_users.py

```diff
@@ -330,7 +330,7 @@ def read(self, fields=None, load='_classic_read'):
 
     @api.model
     def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
-        groupby_fields = set([groupby] if isinstance(groupby, pycompat.string_types) else groupby)
+        groupby_fields = set([x.split(':')[0] for x in ([groupby] if isinstance(groupby, pycompat.string_types) else groupby)])
```

Comment:
Do we need the outer square brackets?

Reply:
We can get away without them. I don't feel there's that much difference between having the brackets there and not having them there. @joe-gb , any thoughts?